docs/how-to-setup-freecodecamp-locally.md

```diff
@@ -63,13 +41,16 @@ That's it, you can now skip to the 'syncing up from parent' section after you ha
 
 ### How to prepare your local machine
 
-Start by installing the prerequisite software for your operating system.
+Here is a minimum system requirement for running freeCodeCamp locally:
+
+- 8 GB RAM
+- Relatively fast CPU (4+ cores)
+- Windows 10 or 11 (with WSL), macOS, or Linux
 
-We primarily support development on Linux and Unix-based systems. Our staff and community contributors regularly work with the codebase using tools installed on Ubuntu and macOS.
+Start by installing the prerequisite software for your operating system.
 
-We also support Windows 10 and 11 via WSL2, which you can prepare by [reading this guide](how-to-setup-wsl.md).
+We primarily support development on Linux and Unix-based systems like Ubuntu and macOS. You can develop on Windows 10 or 11 with WSL2 only. You can follow [this guide](how-to-setup-wsl.md) to set up WSL2. You can't use Command Prompt, Git Bash or PowerShell to run freeCodeCamp natively within windows.
```

Comment:
I am happy with keeping this (these), but there is repetition between this and the `!ATTENTION` above.

Reply:
Yes - it was intentional for people not reading. In the longer term the UX around the guide can be made nicer. I am working on something on the side.